src/main/java/com/epam/page/object/generator/validators/DuplicateTitleInInnerSearchRulesValidator.java

```diff
@@ -34,6 +41,7 @@ public boolean isValid(SearchRule searchRule, ValidationContext validationContex
         Set<String> titles = new HashSet<>();
```

Comment:
Please, add unit test for this method.

Reply:
We have DuplicateTitleInInnerSearchRulesValidatorTest class which contains tests for this method.